### ISSUE TITLE
dummy optimizer out of lr scheduler

### ISSUE THREAD

**Issue by fvarno** [varnio/fedsim#59]
``fedsim.lr_schedulers`` is made to made to modify pytorch lr schedulers s.t. they could optionally either receive an optimizer or an initial leanring rate. We needed this because round to round lr scheduler is not associated with any optimizer. However, since we control the updates of round to round lr scheduler inside the algorithm base class, we can make a dummy classifier with the initial leanring rate and pass that optimizer to the lerning rate scheduler constrcutor. This frees us from redefining pytorch lr schedulers and so we can get rid of ``fedsim.lr_schedulers`` altogether.
